docs/6-algorithms-and-data-structures-part-1/hometask/CircularJobRunner.ts

```diff
@@ -0,0 +1,53 @@
+import { CustomJob } from "./CustomJob";
+import { Job } from "./Job";
+
+export class CircularJobRunner {
+  initialCapacity: number = 10;
+  jobs: Job[] = [];
+
+  constructor() {}
+
+  insert(job: Job) {
+    let contain = false;
+
+    for (let i = 0; i < this.jobs.length; i++) {
+      if (this.jobs[i].priority > job.priority) {
+        this.jobs.splice(i, 0, job);
```

Comment:
please implement jobs using heap data structure that is based on tree (can be linked tree or array representing tree)

Reply:
Just saw your comment, will try to implement this

Reply:
added two variants of binary heap